shared/p2p/BUILD.bazel

```diff
@@ -29,6 +29,10 @@ go_library(
     ],
 )
 
+# by default gazelle tries to add all the test files to the  first
+# go_test; we want to treat feed_concurrent_test.go differently
+# gazelle:exclude feed_concurrent_test.go
```

Comment:
Please move this to be next to the go test on line 61, unless gazelle requires it at the top up here for some reason

Reply:
Sure done, locally `bazel run //:gazelle -- fix` didn't try to change anything.